x-pack/platform/plugins/shared/actions/server/integration_tests/__snapshots__/connector_types.test.ts.snap

```diff
@@ -3154,135 +3154,6 @@ Object {
 }
 `;
 
-exports[`Connector type config checks detect connector type changes for: .workflows 1`] = `
```

Comment:
These deletions seem odd; I think these are almost always additive, and these changes seem to be work a workflow-related connector.  Which I assume is unrelated to your change.

Do you know where this is coming from?